src/qibocal/protocols/characterization/randomized_benchmarking/standard_rb.py

```diff
@@ -156,17 +157,29 @@ def acquire(params: RBParameters, *args) -> RBData:
         if params.noise_model
         else None
     )
+    # TODO extract the noise parameters from the build noise model and add them to params object.
     # Execute the scan.
     data = execute(scan, params.nshots, noise_model)
```

Comment:
Maybe it's just me but I would prefer to not use the `execute()` and have it on the `_acquisition()`. It looks more like the other routines that way. @andrea-pasquale @wilkensJ what's your opinion on this ?

```suggestion
    # 2. Execute the scan.
    data_list = []
    # Iterate through the scan and execute each circuit.
    for circuit in scan:
        # The inverse and measurement gate don't count for the depth.
        depth = (circuit.depth - 2) if circuit.depth > 1 else 0
        if noise_model is not None:
            circuit = noise_model.apply(circuit)
        samples = circuit.execute(nshots=params.nshots).samples()
        # Every executed circuit gets a row where the data is stored.
        data_list.append({"depth": depth, "samples": samples})
```